runtime/sqldelight-runtime/src/main/java/com/squareup/sqldelight/internal/QueryList.kt

```diff
@@ -0,0 +1,29 @@
+package com.squareup.sqldelight.internal
+
+import com.squareup.sqldelight.Query
+
+/**
+ * TODO: Get rid of if CopyOnWriteArrayList joins kotlin.collections?
+ */
+class QueryList {
+  internal var queries: List<Query<*>> = emptyList()
```

Comment:
This can be a `val` if moved to a constructor property and `plus` changed to `= QueryList(queries + other.queries)`